Fix OpenProcessing visualID type mismatch causing rate limiting [2.0]

ksen0 opened the pull request:
Relates to https://github.com/processing/p5.js-website/issues/955
Applies @junseok44 's patch to 2.0 branch

montoyamoraga commented on src/data/data.yml:
```diff
@@ -1,2 +1,2 @@
 title: p5.js
-version: 1.1.9
+version: 1.2.0
```
hi @suhascv,
i don't think this is necessary for the pull request, right?
this file hasn't been updated in 7 months, so i will ask other maintainers what is it :)

suhascv replied:
that's probably not required

montoyamoraga commented on src/data/examples/en/10_Interaction/11_WeightLine.js:
```diff
@@ -0,0 +1,55 @@
+ * @description contributed by <a href="https://www.rit.edu/directory/wmhics-w-michelle-harris">
+   <b>Prof WM Harris,</b></a> using the random function with events to color/weight a line<br/>
+  <b>How</b> to use the random function with events to color/ weight a line
+	dependent on mouse location, lmb clicks, character key types, and
```
could you please change LMB to left mouse button?
it took me a while to understand what was it :)

suhascv replied:
I'll change that

montoyamoraga reviewed (changes requested):
thanks @suhascv i added some of my comments :)

montoyamoraga reviewed (approved):
that's great, thanks!

junseok44 committed:
Fix OpenProcessing visualID type mismatch causing rate limiting

- Change visualID type from string to number to match API response
- Update all OpenProcessing functions to accept number parameters
- Add explicit String() conversion in getStaticPaths for URL compatibility
- Remove unnecessary type conversions in sketch lookups
- Enable proper memoization to reduce API calls and prevent 429 errors
